Index: src/main/java/tornadofx/ViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/tornadofx/ViewModel.kt	(revision 938abc2cdef4e9b10d7733f71b9ee103b2dacaf0)
+++ src/main/java/tornadofx/ViewModel.kt	(revision )
@@ -196,6 +196,11 @@
         validationContext.decorationProvider = decorationProvider
     }
 
+    var onrebind: () -> Unit? = {}
+    fun setOnRebind(rebindFunction: () -> Unit) {
+        onrebind = rebindFunction
+    }
+
     val isValid: Boolean get() = validationContext.isValid
 
     /**
@@ -262,6 +267,7 @@
 
 fun <T : ViewModel> T.rebind(op: (T.() -> Unit)) {
     op()
+    onrebind()
     rollback()
 }
 
@@ -379,7 +385,7 @@
     return null
 }
 
-open class ItemViewModel<T>  : ViewModel() {
+open class ItemViewModel<T> : ViewModel() {
     val itemProperty = SimpleObjectProperty<T>()
     var item by itemProperty
 
\ No newline at end of file
